Mirage-Widget/Assets/MirageWidget/Examples/MirageWidgetExample.cs

```diff
@@ -16,7 +16,10 @@ private void Start()
 			Debug.Log("MirageWidgetExample:: start");
 			_mirageWebViewManager = new MirageWebViewManager();
 			_messageBus.AddListener(_mirageWebViewManager);
-			_mirageWebViewManager.ShowWebView("https://google.com");
+			
+			//TODO ANTON finish this
```

Comment:
anton finish this, or create a task for it

Reply:
Added a ticket name to the comment